Translated/Quests/Story/06 Three Stand Alone/qst_1200060010.csv

```diff
@@ -1,6 +1,6 @@
 name010#0,"""Aina"""
 name020#0,"""Manon"""
-text020#0,"""え？\u3000これもドールズ？ """
-text030#0,"""ではないけれど、このままにはしておけないわね。<br>おそらく、どこかに制御装置があるはずよ。"""
-text040#0,"""危ないし、壊しちゃおっか。"""
-text050#0,"""それでもかまわないけど……。<br>あなた、意外に大胆ね。"""
+text020#0,"""Huh? Are these things DOLLS too?"""
+text030#0,"""No, but we shouldn't leave them as it is. I'd say there's likely a control unit somewhere."""
+text040#0,"""It's dangerous, so let's bust 'em up just in case."""
+text050#0,"""That's fine, too... Quite assertive of you, Aina."""
```

Comment:
This is the Drome cutscene, so it probably is better worded as follows:
"Huh? Is this thing a DOLL too?"
"No, but we shouldn't leave it as it is. I'd say there's likely a control unit somewhere."
"It's dangerous, so let's bust it up just in case."